src/serviceregistry/manual/examples.md

```diff
@@ -0,0 +1,2390 @@
+# **Examples of data transfer objects sent and received**
+
+This file contains specific examples of what payloads are expected and returned by each endpoint maintained by the service registry.
+
+# Discovery endpoints
+
+## **Device discovery**
+- **Register device:** POST /serviceregistry/device-discovery/register
+  
+  Request body: 
+  ~~~
+  {
+    "name": "thermometer1",
+    "metadata": {
+      "scales": ["Kelvin", "Celsius"],
+      "max-temperature": {"Kelvin": 310, "Celsius": 40},
+      "min-temperature": {"Kelvin": 260, "Celsius": -10},
+      "appliable": ["spring", "autumn"]
+    },
+    "addresses": [
+      {
+        "type": "MAC",
+        "address": "81:ef:1a:44:7a:ff"
+      }
+    ]
+  }
```

Comment:
After my current PR are merged, the input here just a string list without the types.